src/gui/render/array_buffer/star_data.cpp

```diff
@@ -12,9 +11,21 @@
 
 namespace gui {
 
-namespace data_structures {
+namespace array_buffer {
 
-SkyData::SkyData(std::filesystem::path path) {
+StarShape::StarShape() :
+    shape_buffer_(ArrayBuffer(std::vector<glm::vec2>({
+        {0,  1 },
+        {1,  0 },
+        {-1, 0 },
+        {0,  -1}
+}))) {
```

Comment:
Add an initializer list ctor to ArrayBuffer

This calls **four** constructors instead of two

1. `std::initializer_list`
2. `std::vector`
3. `ArrayBuffer`
4. `ArrayBuffer`